tests/python/dashboard/utils.py

```diff
@@ -94,6 +94,23 @@ class DashboardTester:
     def __init__(self, sb):
         self.sb = sb
 
+        # Set up the examples directory path once
+        impactx_directory = Path(get_impactx_root_dir())

```

Comment:
Unrelated to this PR:
It would be useful to generalize `get_impactx_root_dir` to also work with an installed ImpactX (after `pip_install` and with no original "source" dir present).

For that, I need to make sure we actually install the examples & tests (add control in the build system).